.jvmopts

```diff
@@ -1,3 +1,9 @@
 -Xss16M
 -Xmx4G
 -XX:+UseCompressedOops
+--add-exports=jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED
+--add-exports=jdk.compiler/com.sun.tools.javac.code=ALL-UNNAMED
+--add-exports=jdk.compiler/com.sun.tools.javac.file=ALL-UNNAMED
+--add-exports=jdk.compiler/com.sun.tools.javac.parser=ALL-UNNAMED
+--add-exports=jdk.compiler/com.sun.tools.javac.tree=ALL-UNNAMED
+--add-exports=jdk.compiler/com.sun.tools.javac.util=ALL-UNNAMED
```

Comment:
`javafmt` used to work without these before because `googlejavaformat` classes used to be part of the sbt Jar. Without these options, the javafmt fails with:
```
[error] (Compile / javafmtCheck) java.lang.IllegalAccessError: class com.google.googlejavaformat.java.ImportOrderer (in unnamed module @0x47ef13b2) cannot access class com.sun.tools.javac.parser.Tokens$TokenKind (in module jdk.compiler) because module jdk.compiler does not export com.sun.tools.javac.parser to unnamed module @0x47ef13b2
```

Note that this is a pretty standard requirement. For example, when one wants to use Google java format plugin in IntelliJ, you need to add these options as well.

Reply:
Can we instead `--limit-modules` to **not contain** `javac` and put Frgaal on the classpath instead? Unless we do it, we'll continue to have inconsistency between the syntax we use and the syntax JDK's compiler supports.

Reply:
@JaroslavTulach Is this even possible? To instruct `sbt` to start with different `--class-path`? Note that this file, `.jvmopts`, applies only to how `sbt` starts itself, not how it invokes any subprocesses.

Reply:
> and put Frgaal on the classpath

I meant: load the Frgaal with the same classloader as the google-java-format. That's certainly possible. `.jvmopts` would only be used to `--limit-modules` and make sure Javac isn't present.